Otec/NewsRepository.swift

```diff
@@ -60,7 +60,7 @@ class NewsRepository: CRUDRepository {
     }
     
     private func urlStringWithPath(path: String) -> String {
-        return "http://43e9c7c6.ngrok.io" + "/" + path
+        return Constants.CanillitaBackendURL + "/" + path
```

Comment:
⚠️ Base path URL should not be added at this point. The `NSURLSessionBackend` compounds the path by adding the `basePath` you've passed in the configuration object.